Automattic/wp-calypso
# My Home: Add new stats card for experimental layout

#### Changes proposed in this Pull Request

Adds a new version of the "Stats at a glance" card to the experimental home layout.

Site | Desktop | Mobile
--- | --- | ---
Unlaunched | ![image](https://user-images.githubusercontent.com/1233880/80227487-f6d55000-864d-11ea-9502-7eb006ae6e00.png) | ![image](https://user-images.githubusercontent.com/1233880/80227497-fb9a0400-864d-11ea-9e6d-ef0b841f507a.png)
Site with no traffic | ![image](https://user-images.githubusercontent.com/1233880/80387896-7f036180-88a9-11ea-8c9f-2d2c960d90cf.png) | ![image](https://user-images.githubusercontent.com/1233880/80387917-8591d900-88a9-11ea-8ba5-c09163031d1c.png)
Site with traffic | ![image](https://user-images.githubusercontent.com/1233880/80387943-8e82aa80-88a9-11ea-8002-defb7dd0c8e4.png) | ![image](https://user-images.githubusercontent.com/1233880/80387971-97737c00-88a9-11ea-8149-9c4f2c5f6e3d.png)


#### Testing instructions

* Load the experimental home layout: `/home/:site/?flags=home/experimental-layout` for an unlaunched site.
* Make sure the stats card is displayed with a placeholder that sets expectations for this section.
* Make sure the "Learn about stats" link opens the stats support doc in a dialog.
* Switch to a launched site with traffic.
* Make sure the stats cards displays a chart with the traffic of the last week (last 7 days).
* Make sure the most popular day and time of the previous 7 days is displayed below the chart.
* Make sure the card displays the title the most visited post during the previous 7 days.
* Make sure the card displays the title the most visited page during the previous 7 days.
* Make sure the total views of the last week are displayed if no posts **OR** pages were visited during the previous 7 days.
* Make sure the total visitors of the last week are displayed if no posts **AND** pages were visited during the previous 7 days.
* Switch to a launched site without traffic.
* Make sure the stats card is displayed with a placeholder that includes a snippet educating users about stats.
* Make sure the "Read more" link opens the stats support doc in a dialog.

Fixes #41034
